pysensors/optimizers/_gqr.py

```diff
@@ -29,7 +29,7 @@ class GQR(QR):
 
```

Comment:
This is a generalized version of cost constraints
    in the sense that users can allow n constrained sensors in the constrained area.
    if n = 0 this converges to the CCQR results.

should be changed to QR